azure/services/availabilitysets/client.go

```diff
@@ -30,7 +30,7 @@ import (
 
 // Client wraps go-sdk.
 type Client interface {
-	Get(ctx context.Context, spec azure.ResourceSpecGetter) (interface{}, error)
+	Get(ctx context.Context, spec azure.ResourceSpecGetter) (result interface{}, err error)
 	CreateOrUpdateAsync(context.Context, azure.ResourceSpecGetter, interface{}) (interface{}, azureautorest.FutureAPI, error)
```

Comment:
@CecileRobertMichon Should the named return be added in the declarations for `CreateOrUpdateAsync`, `DeleteAsync`, and `IsDone` as well?

Reply:
They should, good catch thanks!